.gitignore

```diff
@@ -1,4 +1,5 @@
 _build
 .merlin
 .DS_Store
+node_modules
```

Comment:
Why is there a change in the gitignore here?

Reply:
I'm using a vs code  plugin([reason-language-server](https://github.com/jaredly/reason-language-server)) that support OCaml and ReasonMl. That plugin uses node_modules to store some information. Keep it maybe benefit many other vs code users or remove it?

Reply:
Got it.